queue/message_prioritizer_test.go

```diff
@@ -0,0 +1,252 @@
+package queue
```

Comment:
spec tests not unit tests

Reply:
wdym?

Reply:
We use spec tests for the spec not unit tests (you can do both but we need spec tests so we can test it out in the implementation)

Reply:
> 

thx, @nivBlox is helping me with this

Reply:
Done